src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisController.java

```diff
@@ -843,7 +861,7 @@ private void tree(AnalysisSubmission submission, Model model) throws IOException
 				model.addAttribute("newick", tree);
 
 				// inform the view to display the tree preview
-				model.addAttribute("preview", "tree");
```

Comment:
Let's talk about this when you get in.

Reply:
Removed the call to this method in 6b86f7ebf4835ce518b8f550a8499fecd55c59b2. Will discuss and add back in when working on the tree view.